cluster-operator/src/test/java/io/strimzi/operator/cluster/model/TopicOperatorTest.java

```diff
@@ -196,4 +199,17 @@ private void assertLoggingConfig(Deployment dep) {
         assertEquals("topic-operator-metrics-and-logging", volume.getName());
         assertEquals("foo-topic-operator-config", volume.getConfigMap().getName());
     }
+
+    @After
+    public void cleanUp() {
```

Comment:
This is basically exactly the same code as before: You should factor it into a method which you can reuse.

Reply:
I wonder how expensive it is. Would it make sense to have simply some abstract class with this method?